04.class/lib/view.js

```diff
@@ -0,0 +1,33 @@
+import enquirer from "enquirer";
+
+export class View {
+  printMemos(memos) {
+    memos.forEach((memo) => {
+      console.log(memo.title);
+    });
+  }
+
+  displayMemoSelectionPrompt(memos, act) {
+    const choices = memos.map((memo) => {
+      return { name: memo.title, value: memo.rowid };
+    });
+    const prompt = new enquirer.Select({
+      name: "memo",
+      message: `Choose a memo you want to ${act}:`,
+      choices: choices,
```

Comment:
```suggestion
      choices,
```

でも動きそう（が、formatterがこう修正したのかな？）

Reply:
修正しました。
（プロパティ名を省略するのとしないのとを同居させられることを知りませんでした）